# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../strace/arch-3-avr32-hotfix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./linux/avr32/syscallent.h.vanilla	2007-09-04 11:46:26.000000000 +0200
+++ ./linux/avr32/syscallent.h	2007-09-04 11:46:38.000000000 +0200
@@ -292,7 +292,6 @@
 	{ 4,	0,	printargs,		"sync_file_range" }, /* 262 */
 	{ 4,	0,	printargs,		"tee"		}, /* 263 */
 	{ 4,	0,	printargs,		"vmsplice"	}, /* 264 */
-	{ 6	0,	printargs,		"sys_epoll_pwait" }, /* 265 */
 	{ 2,	TI,	sys_msgget,		"msgget"	}, /* 266 */
 	{ 4,	TI,	sys_msgsnd,		"msgsnd"	}, /* 267 */
 	{ 5,	TI,	sys_msgrcv,		"msgrcv"	}, /* 268 */
